extensions/adobe/experience/adcloud/experienceevent.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/experienceevent",
+  "title": "Adobe AdCloud ExperienceEvent Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Adobe Campaign extension to ExperienceEvent.",
+  "definitions": {
+    "experienceevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/adcloud/eventType": {
+          "title": "Event Type",
+          "type": "string",
+          "description": "Adobe AdCloud event type."
+        },
+        "https://ns.adobe.com/experience/adcloud/marketing/campaignDetails": {
+          "title": "Ad Campaign Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/campaignDetails",
+          "description":
+          "Adobe AdCloud ad campaign hierarchy details."
+        },
+        "https://ns.adobe.com/experience/adcloud/marketing/inventoryDetails": {
+          "title": "Ad Inventory Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/inventoryDetails",
+          "description":
+          "Adobe AdCloud inventory details."
+        },
+        "https://ns.adobe.com/experience/adcloud/marketing/conversionDetails": {
+          "title": "Ad Conversion Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/conversionDetails",
+          "description":
+          "Adobe AdCloud ad conversion hierarchy details."
+        },
+        "https://ns.adobe.com/experience/adcloud/media/advertisementDetails": {
+          "title": "Ad Asset Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/advertisementDetails",
+          "description":
+          "Digital Advertisement Asset Details"
```

Comment:
Why do you need the "digital advertisement" reference?
And why isn't this an actual xdm:asset here?

Reply:
advertisement-details schema contains a field which is referencing "https://ns.adobe.com/xdm/assets/aggregated-asset" to take advantage of all existing asset fields. The schema contains ad specific fields which is not captured in asset as of today. It does not extend, but wraps the asset schema. Open to creating an asset extension instead if deemed better suited.

Reply:
AFAIK, there is no such thing called an "aggregated asset" - where did that come from?   What do you mean it wraps but doesn't extend?

Reply:
the problem, however, is that we won't be making that schema stable(izing) any time soon - so if that is important we need to resolve that.  @trieloff